CheckDrive.Api/CheckDrive.Application/Services/Authorization/AuthService.cs

```diff
@@ -0,0 +1,37 @@
+﻿using CheckDrive.Application.DTOs.Identity;
+using CheckDrive.Application.Interfaces.Authorization;
+using CheckDrive.Domain.Exceptions;
+using Microsoft.AspNetCore.Identity;
+
+namespace CheckDrive.Application.Services.Authorization;
+
+public sealed class AuthService : IAuthService
+{
+    private readonly IJwtTokenGenerator _jwtTokenGenerator;
+    private readonly UserManager<IdentityUser> _userManager;
+
+    public AuthService(
+        IJwtTokenGenerator jwtTokenGenerator,
+        UserManager<IdentityUser> userManager)
```

Comment:
If there are more than 2 parameters you can put them on separate lines, if not use the same line